flamedisx/xenon/x1t_sr1.py

```diff
@@ -68,9 +84,19 @@ def read_maps_tf(path_bag, is_bbf=False):
         yy_ref_bag.append(tf.convert_to_tensor(tmp['map'], dtype=fd.float_type()))
         data_bag.append(tmp)
     domain_def = tmp['coordinate_system'][0][1]
-
+    
```

Comment:
Blank line with whitespace (not sure why codefactor is sleeping on the job)

Reply:
I will let codefactor catch me in the future!